app/assets/stylesheets/application.css.scss

```diff
@@ -779,5 +779,14 @@ font-size:6px;
 
 }
 
+textarea#data_source_dictionary {
+  width: 100%;
+}
+
+.data_source .field input
+{
+  width: 50%;
+}
+
```

Comment:
Better to put this in [customization.css.scss](https://github.com/GovWizely/webservices/blob/master/app/assets/stylesheets/customizations.css.scss). The contents of this file are copied verbatim from developerportal; any webservices-specific modifications are in customizations.css.scss. This way, if developerportal CSS is updated we can just copy it here without worrying about deleting modifications. Not a great system but that's what we have.